data/ecosystems/c/celo.toml

```diff
@@ -7954,3 +7955,63 @@ url = "https://github.com/koshikraj/safe2link"
 
 [[repo]]
 url = "https://github.com/anubhavitis/Eth-Fit"
+
+[[repo]]
+url = "https://github.com/smartcontractkit/chainlink"
+
+[[repo]]
+url = "https://github.com/thirdweb-dev/js"
+
+[[repo]]
+url = "https://github.com/thirdweb-dev/docs"
+
+[[repo]]
+url = "https://github.com/thirdweb-dev/docs-v2"
+
+[[repo]]
+url = "https://github.com/thirdweb-dev/contracts"
+
+[[repo]]
+url = "https://github.com/thirdweb-dev/unity-sdk"
+
+[[repo]]
+url = "https://github.com/pyth-network/pyth-crosschain"
+
+[[repo]]
+url = "https://github.com/pyth-network/pyth-observer"
+
+[[repo]]
+url = "https://github.com/pyth-network/documentation"
+
+[[repo]]
+url = "https://github.com/pyth-network/grant-program"
+
+[[repo]]
+url = "https://github.com/pyth-network/pyth-gitbook"
+
+[[repo]]
+url = "https://github.com/pyth-network/governance-ui"
+
+[[repo]]
+url = "https://github.com/centrifuge/centrifuge-chain"
+
+[[repo]]
+url = "https://github.com/centrifuge/documentation"
+
+[[repo]]
+url = "https://github.com/centrifuge/chainbridge-ui"
+
+[[repo]]
+url = "https://github.com/centrifuge/slips"
+
+[[repo]]
+url = "https://github.com/centrifuge/Grants-Program"
+
+[[repo]]
+url = "https://github.com/centrifuge/apps"
+
+[[repo]]
+url = "https://github.com/centrifuge/tinlake-apps"
+
+[[repo]]
+url = "https://github.com/pimlicolabs/alto"
```

Comment:
Please remove these, the subecosystems connections will pick them on our internal rollups ETL